package-overrides/npm/aws-sdk@2.4.0.json

```diff
@@ -0,0 +1,8 @@
+{
+  "main": "dist/aws-sdk",
+  "format": "global",
+  "exports": "AWS",
```

Comment:
This isn't a valid property - it should either be jspm 0.17 metadata:

```javascript
  "meta": {
    "dist/aws-sdk.js": { "exports": "AWS" }
  }
```

otherwise it's probably best to use jspm 0.16 compatible shim:

```json
  "shim": {
    "dist/aws-sdk": { "exports": "AWS" }
  }
```

automatic detection applies though when neither is present, which is also ok.